cards/document-standard/template.hbs

```diff
@@ -0,0 +1,96 @@
+<div class="HitchhikerDocumentStandard {{cardName}}">
+  {{> image }}
+  <div class="HitchhikerDocumentStandard-body">
+    {{> title }}
+    {{> subtitle }}
+    <div class="HitchhikerDocumentStandard-contentWrapper">
+      <div class="HitchhikerDocumentStandard-info">
+        {{> details }}
+      </div>
+      {{> ctas }}
+    </div>
+  </div>
+</div>
+
+{{#*inline 'image'}}
+{{#if card.image}}
+<div class="HitchhikerDocumentStandard-imgWrapper">
+  <img class="HitchhikerDocumentStandard-img" src="{{#unless (isNonRelativeUrl card.image)}}{{@root.relativePath}}/{{/unless}}{{card.image}}" alt="{{#if card.altText}}{{card.altText}}{{/if}}"/>
+</div>
+{{/if}}
+{{/inline}}
+
+{{#*inline 'title'}}
+{{#if card.title}}
+<div class="HitchhikerDocumentStandard-title">
+  {{#if card.url}}
+  <a class="HitchhikerDocumentStandard-titleLink js-HitchhikerDocumentStandard-titleLink"
+     href="{{#unless (isNonRelativeUrl card.url)}}{{@root.relativePath}}/{{/unless}}{{card.url}}"
+     data-eventtype="TITLE_CLICK"
+     data-eventoptions='{{json card.titleEventOptions}}'
+     target={{#if card.target}}"{{card.target}}"{{else}}"_top"{{/if}}>
+    {{card.title}}
+  </a>
+  {{else}}
+  {{card.title}}
+  {{/if}}
+</div>
+{{/if}}
+{{/inline}}
+
+{{#*inline 'subtitle'}}
+{{#if card.subtitle}}
+<div class="HitchhikerDocumentStandard-subtitle">
+  {{card.subtitle}}
+</div>
+{{/if}}
+{{/inline}}
+
+{{! Displays the details for the card. If showMoreDetails has been configured,
```

Comment:
Let's remove references to `showMoreDetails` here.

Reply:
ah missed this!